dbscan/rowscanner_test.go

```diff
@@ -22,13 +22,22 @@ type JSONObj struct {
 	Key string
 }
 
+type DeepNested struct {
+	NestedLevel0
+}
+
+type NestedLevel0 struct {
+	NestedLevel1
+}
+
 type NestedLevel1 struct {
 	NestedLevel2
 	Nested2 NestedLevel2
 }
 
 type NestedLevel2 struct {
 	Foo string
+	Bar string
 }
```

Comment:
Why not to just add `NestedLevel3` and `NestedLevel4` instead of `DeepNested` and `NestedLevel0`.
```suggestion
type NestedLevel1 struct {
	NestedLevel2
	Nested2 NestedLevel2
}

type NestedLevel2 struct {
      NestedLevel3
	Foo string
}

type NestedLevel3 struct {
      NestedLevel4
}

type NestedLevel4 struct {
      DeepNested1 string
      DeepNested2 string
}
```

Reply:
was going for lower impact to the existing tests, but I will commit this change instead, yes.